LocalWiki.php

```diff
@@ -27,6 +27,39 @@ function onBeforePageDisplay( OutputPage $out ) {
 			$out->addMeta( 'og:image:width', '1200' );
 		}
 
+		break;
+	case 'gratisdatawiki':
+		$wgHooks['overridePageMetaTags'][] = 'onOverridePageMetaTags';
```

Comment:
Where does this hook come from? I can't find any hook with this name anywhere.

Reply:
> Where does this hook come from? I can't find any hook with this name anywhere.

Let me link you https://github.com/wikimedia/Wikibase/blob/master/repo/includes/Actions/ViewEntityAction.php#L140

Reply:
> 

There are no replacements for some stuff.

Reply:
Um... that is not a hook.

Reply:
> Um... that is not a hook.

Yeah, that's a function, but I thought after seeing a function, knowing the hook shouldn't be hard.... (dumb me)